caffe2/core/tensor.h

```diff
@@ -53,6 +53,9 @@ class CAFFE2_API Tensor final {
   explicit Tensor(Storage storage)
       : impl_(c10::make_intrusive<TensorImpl, UndefinedTensorImpl>(std::move(storage))) {}
 
+  explicit Tensor(DeviceType device_type)
```

Comment:
this is necessary?

Reply:
Yes, I accidentally deleted it in a previous PR (but it still worked because DeviceType implicitly converted into Storage).